From 124f957514ea57d1fe9ebcb6173dac54545f62a6 Mon Sep 17 00:00:00 2001
From: afresh1 <afresh1@openbsd.org>
Date: Wed, 18 Oct 2023 01:49:26 +0000
Subject: [PATCH] Increase max depth for termcaps

With the update to ncurses 6.4-20230826 /etc/termcap now needs at
least a 36 deep search.  Instead, we'll just bump to 64.  It seems
32 has been enough since perl 5.001 in 1995, so hopefully this buys
us at least another 28 years.

"please commit that" deraadt@
---
 gnu/usr.bin/perl/cpan/Term-Cap/Cap.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnu/usr.bin/perl/cpan/Term-Cap/Cap.pm b/gnu/usr.bin/perl/cpan/Term-Cap/Cap.pm
index 12d8299a3a4c..4ad9dd21e0ab 100644
--- cpan/Term-Cap/Cap.pm.orig
+++ cpan/Term-Cap/Cap.pm
@@ -280,7 +280,7 @@ sub Tgetent
 
     $first = 0;    # first entry (keeps term name)
 
-    $max = 32;     # max :tc=...:'s
+    $max = 64;     # max :tc=...:'s
 
     if ($entry)
     {
